app/static/js/createEvents.js

```diff
@@ -8,14 +9,15 @@ function updateDate(obj) {
   var newMonth = dateToChange.getMonth();
   var newYear = dateToChange.getFullYear();
   var newDay = dateToChange.getDate();
-  if(obj.id == "startDatePicker") {
-    $("#endDatePicker").datepicker({minDate: new Date(  newYear, newMonth, newDay)});
-    $("#endDatePicker").datepicker( "option", "minDate", new Date(  newYear, newMonth, newDay));
+  if(obj.className.includes("startDatePicker")) {
+    
+    $("#endDatePicker-"+$(obj).data("page-location")).datepicker({minDate: new Date(  newYear, newMonth, newDay)});
+    $("#endDatePicker-"+$(obj).data("page-location")).datepicker( "option", "minDate", new Date(  newYear, newMonth, newDay));
```

Comment:
Formatting on this line is a little wacky.

Reply:
Fixed